src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,130 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private long id;
+
+	@Column(nullable = false)
```

Comment:
According to the card specification, genre can be nullable. The other fields should not be nullable, but this one can be set to null.